.github/workflows/rosa-run-benchmark.yml

```diff
@@ -0,0 +1,90 @@
+name: ROSA Cluster - Run Benchmark
+
+on:
+  workflow_dispatch:
+
+concurrency: shared_cluster
+
+jobs:
+
+  delete:
+    name: Run Benchmark
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Set up JDK
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: '11'
+          cache: 'maven'
+
+      - name: Cache Maven Wrapper
+        uses: actions/cache@v3
+        with:
+          path: |
+            .mvn/wrapper/maven-wrapper.jar
+          key: ${{ runner.os }}-maven-wrapper-${{ hashFiles('**/maven-wrapper.properties') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-wrapper-
+
+      - name: Setup ROSA CLI
+        uses: ./.github/actions/rosa-cli-setup
+        with:
+          aws-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-default-region: ${{ vars.AWS_DEFAULT_REGION }}
+          rosa-token: ${{ secrets.ROSA_TOKEN }}
+
+      - name: Setup Go Task
+        uses: ./.github/actions/task-setup
+
+      - name: Login to OpenShift cluster
+        run: ./rosa_oc_login.sh
+        working-directory: provision/aws
+
+      - name: Build with Maven
+        run: |
+          ./mvnw -B clean install -DskipTests
+          tar xfvz benchmark/target/keycloak-benchmark-0.10-SNAPSHOT.tar.gz
```

Comment:
I see there is no regex for benchmark version, would this be handled as a form input later on ? Otherwise it would break when we upgrade version within the project.

Reply:
Was being a little bit lazy there perhaps ;)

See below I can just replace it with a '*' I reckon